src/auth-service/routes/v2/preferences.js

```
@@ -104,6 +104,82 @@ function validateSelectedSitesField(value) {
+  }
+
+  const numericValid = validateNumericFields([
+    "latitude",
+    "longitude",
+    "approximate_latitude",
+    "approximate_longitude",
+  ]);
+
+  const stringValid = validateStringFields(["name", "search_name"]);
+
+  const tags = value && value.site_tags;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)